BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -131,6 +136,39 @@ public struct AccessibilityElement: Element {
     }
 }
 
+extension AccessibilityElement.Trait: Hashable, Equatable {
+    /// This conformance to `Hashable` is provided to allow traits to be included in a `Set`.
+    /// - Important: ⚠️ This implementation does not take equality of associated values on `.adjustable` into account.  ⚠️
```

Comment:
This conformance is public, right? We should probably also get this note in the public API for this type.

Reply:
It was previously hashable as well. This isn't a change in that regard. I just moved the conformance to the extension.

Reply:
It was previously hashable as well. This isn't a change in that regard. I just moved the conformance to the extension.